examples/dataflow-python-examples/dataflow_python_examples/datastore_schema_import.py

```diff
@@ -53,7 +53,7 @@ def main():
     client = datastore.Client()
 
     filename = args.schema_file
-    print('Processing file %s' % filename)
+    print(('Processing file %s' % filename))
     csvfile = open(filename, 'r')
     table = os.path.splitext(os.path.basename(filename))[0]
```

Comment:
This is used by the code lab / README to populate schema data in datastore that is dynamically read at runtime by the configurable data pipeline.

This could easily be moved to a scripts / utils dir no other code in this dir directly relies on it.

Reply:
we will do this after final directory restructuring task